.github/labels.yml

```diff
@@ -229,25 +218,13 @@
 - name: hint/needs-community-input
   color: 0623cc
   description: "Hint: Needs Community Input"
-- name: hint/needs-review
-  color: 0623cc
-  description: "Hint: Needs Review"
-- name: hint/needs-help
-  color: 0623cc
-  description: "Hint: Needs Help"  
 - name: hint/needs-docs
   color: 0623cc
   description: "Hint: Needs Docs"
 
 ###
 ### Statuses
 ###
-- name: status/done
-  color: edb3a6
-  description: "Status: Done"
-- name: status/approved-waiting
-  color: edb3a6
-  description: "Status: Approved and Awaiting Merge"
```

Comment:
Since we're removing the "Done" and "Approved" labels, is the following the desired flow:
- I merge my PR into the master branch
- Instead of changing the status of the issue, I simply close the issue with a comment (referencing the PR)

Reply:
I think your intuition is right here and I like this flow better 👍

Reply:
You can also add a tag in your PR,
"fixes #issue-id" will autoclose the issue on merge.
"contributes to #issue-id" is also useful on larger tasks (usually a smell that your issue is too big though).

Reply:
Great. I think that's the way to go - add "fixes #issue" to auto-close it, and leave a description in the issue if further clarification is necessary.